model-optimizer/extensions/front/onnx/mask_rcnn_conversion.py

```diff
@@ -92,8 +91,11 @@ def insert_do(graph: Graph, replacement_descriptions):
     do_node.out_port(1).get_connection().insert_node(Cast(graph, {'dst_type': np.int64}).create_node())
 
 
-def insert_ExperimentalDetectronROIFeatureExtractor1(graph: Graph):
-    old_output_node = Node(graph, '6795')
+def insert_ExperimentalDetectronROIFeatureExtractor1(graph: Graph, replacement_descriptions: dict):
+    if 'ROIFeatureExtractor1_output' not in replacement_descriptions:
```

Comment:
Please, add a comment that this is a case of Faster-RCNN and this is expected that attribute is not set.